config/initializers/rack_attack.rb

```diff
@@ -256,4 +257,28 @@ def headers
   user = req.env['warden'].user || AnonymousUser.new
   analytics = Analytics.new(user: user, request: req, session: {}, sp: nil)
   analytics.rate_limit_triggered(type: req.env['rack.attack.matched'])
+
+  pp payload
+
+  # How can we access Attempts Api Tracker from here?
+  # The user is not logged in yet.
+  # Can we "make" a new user from just their email address?
+
+  if req.env["rack.attack.matched"]
+    @irs_attempts_api_tracker ||= IrsAttemptsApi::Tracker.new(
+      session_id: "",
+      request: req,
+      user: nil,
+      sp: "",
+      cookie_device_uuid: "",
+      sp_request_uri: "",
+      enabled_for_session: "",
+      analytics: analytics,
+    )
+
+    @irs_attempts_api_tracker.login_rate_limited(
+      email: req[:user]["email"],
+    )
+  end
+
```

Comment:
What would be a better way to do this? How can we get an instance of the attempts api tracker inside this listener?

Reply:
I put a message [here](https://gsa-tts.slack.com/archives/C03N6KQBAKS/p1675287724391969?thread_ts=1674249851.206159&cid=C03N6KQBAKS), but the short version is I don't think we can and I think we have other events that should mitigate the need for tracking this.